src/app/share/views/SharedGuestSignUp/ShareGuestSingUpView.tsx

```diff
@@ -22,13 +22,13 @@ import { MAX_PASSWORD_LENGTH } from '../../../shared/components/ValidPassword';
 import { decryptPrivateKey } from 'app/crypto/services/keys.service';
 import TextInput from 'app/auth/components/TextInput/TextInput';
 import PasswordInput from 'app/auth/components/PasswordInput/PasswordInput';
-import Button from 'app/auth/components/Button/Button';
 import { useSignUp } from 'app/auth/components/SignUp/useSignUp';
 import shareService from 'app/share/services/share.service';
-import bigLogo from 'assets/icons/big-logo.svg';
+import { ReactComponent as InternxtLogo } from 'assets/icons/big-logo.svg';
 import { useSelector } from 'react-redux';
 import { RootState } from 'app/store';
 import ExpiredLink from 'app/shared/views/ExpiredLink/ExpiredLinkView';
+import Button from 'app/shared/components/Button/Button';
 
 function ShareGuestSingUpView(): JSX.Element {
```

Comment:
Does the ShareGuestSingUp view not have to redirect to the screen to download the backup key?

Reply:
Mmm not sure because it has another flow. Maybe @aboredvaro can answer that.

Reply:
Yep, after sign up, we should show the "download backup key" view.
I'm aware there are 2 views for signing up right now:
- Sign up
- Invited user Sign up

Please, consider merging it into only 1 signup with parameters so we don't need to maintain 2 views (both for visuals and logic) for the same purpose 🙏🏻

Reply:
Hey @aboredvaro. Thanks to help us with this issue.

I've updated the ShareGest view to allow the new user to download the Backup key.

Can you check the design, in case we missed something?

[Preview Link](https://drive-hy1qjcs0h-internxt-web-team.vercel.app/login)

Reply:
Everything is good, only add a 16px space where the red line is 🙏🏻
<img width="939" alt="Screenshot 2024-02-07 at 16 52 18" src="https://github.com/internxt/drive-web/assets/34165099/6901d709-b094-4aac-bd69-b86ad8678097">

Reply:
Okay, done. I've added the padding. In case you want to check it again @aboredvaro, [here](https://drive-12ln1o9op-internxt-web-team.vercel.app/login).